src/plugins/enzymes/src/DigestSequenceDialog.cpp

```diff
@@ -157,9 +157,11 @@ void DigestSequenceDialog::accept() {
     for (int row = 0; row < itemCount; ++row) {
         const QString &annEncoded = conservedAnnsWidget->item(row)->text();
         QStringList annData = annEncoded.split(" ");
-        assert(annData.size() == 2);
-        QString aName = annData.at(0);
-        QString locationStr(annData.at(1));
+        assert(annData.size() >= 2);
```

Comment:
We do not use asserts anymore, we use SAFE_POINTS

Reply:
Removed at all